DeanerySystem/DeanerySystem.DataAccess/Utilities/DataBaseUtilities.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Linq;
+using DeanerySystem.DataAccess.Concrete;
+using DeanerySystem.DataAccess.DataFeeders;
+using DeanerySystem.DataAccess.Entities;
+
+namespace DeanerySystem.DataAccess.Utilities
+{
+    public static class DataBaseUtilities
+    {
+        public static void FeedDataBase(DeaneryDbContext context)
+        {
+            try
+            {
+                using (var uow = new UnitOfWork(context))
+                {
+                    if (uow.UniversityRepository.Get().Any())
+                    {
+                        return;                       
+                    }
+
+                    var university = new University() { Name = "ЛНУ імені Івана Франка" };
+                    uow.UniversityRepository.Insert(university);
+                    uow.Save();
+
+                    var fDataFeeder = new FacultyDataFeeder(uow);
+                    fDataFeeder.Data.ForEach(f =>
+                    {
+                        f.Id = 0;
```

Comment:
Why should the `Id` be set to `0`?

Reply:
Because it is configured as Identity property with ValueGeneratedOnAdd and conflicts with not 0 Ids. 
Afterward, I have discovered the modern way of mocking data.